### DISCUSSION TITLE
Robotron EC 1835 Turbo Support

### DISCUSSION THREAD

**Discussion by mmoeller86** [86Box/86Box#3337] [Ideas]
I request to add support for the Robotron EC 1835 Turbo Computer.
It is based on the 80386 Processor and is compatible to a western 386-Computer.

**Comment by mmoeller86** [comment]
There maybe existed special versions with 486+ (80486 or Intel Pentium) which
also be included.

**Comment by asiekierka** [comment]
Do you know anyone who actually has the physical machine? What few sources I can find imply that it is "extinct"; if no copies of the ROMs, software, or hardware schematics exist, it cannot be reliably implemented.
